orca-core/orca-core.gradle

```diff
@@ -40,6 +40,8 @@ dependencies {
   compile "com.fasterxml.jackson.module:jackson-module-kotlin:${spinnaker.version('jackson')}"
   compile "org.apache.commons:commons-lang3:3.7"
   compile "de.huxhorn.sulky:de.huxhorn.sulky.ulid:8.1.1"
+  compile "org.springframework.security:spring-security-web:4.2.10.RELEASE"
+  compile "javax.servlet:javax.servlet-api:4.0.1"
```

Comment:
Is this dependency used?

Reply:
Code would not compile without it:

> Task :orca-core:compileJava FAILED
/usr/local/google/home/joonlim/dev/spinnaker/orca/orca-core/src/main/java/com/netflix/spinnaker/orca/config/UserConfiguredUrlRestrictions.java:145: error: cannot access HttpServletRequest
        if (ipMatcher.matches(u.getHost())) {
                     ^
  class file for javax.servlet.http.HttpServletRequest not found
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error